tools/cli/pkg/ers/command/migrate_all.go

```diff
@@ -98,17 +109,38 @@ func (c *MigrationAllCommand) Run() error {
 		c.log.Debugf("Read: %s\n", instance)
 		c.log.Infof("Passing instance %s to workers", instance.Name)
 
-		if !instance.Migrated {
-			payloads <- ers.Work{instance, 0, 0, 1}
-			c.wg.Add(1)
-			c.stats.Add()
-		} else {
-			c.log.Infof("%sInstance %s already migrated%s",
-				Green, instance.Name, Reset)
+		meta, err := c.metadataStorage.Get(instance.Id)
+		if err != nil {
+			c.log.Warnf("Get %s Metadata error: %s", instance.Id, err.Error())
+			continue
+		}
+		if meta.KymaMigrated {
```

Comment:
do not run migration, if it was already done (information stored in local storage)